### PR TITLE
Adding a extra links tab on the header

### PR DESCRIPTION
**Pull request by mla04762**
#1035

### CODE REVIEW

**Comment by mla04762** [comment]
I am currently trying different things to see what would be the best option for having an extra links tab/page. For the most recent commit, I got rid of the links tab from the profile drop down after moving all the links in there to the new extra links tab on the header. Having an extra tab on the header made the header look a bit crowded, so I moved the involvements page in the links tab (people thought that was the least visited tab).

**Comment by amos-cha** [comment]
@mla04762  remember to handle your merge conflicts

File: .env.development [hunk 1]
```diff
@@ -7,4 +7,4 @@
  VITE_API_URL=https://360ApiTrain.gordon.edu/
 
 # @LOCALHOST
-# VITE_API_URL=http://localhost:51626/
+ #VITE_API_URL=http://localhost:51634/
```
Comment (amos-cha) [inline .env.development:10]: revert

File: src/components/Profile/index.jsx [hunk 2]
```diff
@@ -58,6 +59,12 @@ const Profile = ({ profile, myProf }) => {
         </Grid>
       )}
 
+      {/* {((!myProf && profileIsStudent) || canReadStudentSchedules) && (
+        <Grid item xs={12} lg={10} align="center">
+          <SchedulePanel profile={profile} myProf={myProf} isOnline={isOnline} />
+        </Grid>
+      )} */}
+
```
Comment (amos-cha) [inline src/components/Profile/index.jsx:67]: remove if unused

**Review by amos-cha — CHANGES_REQUESTED** [review]

**Comment by russtuck** [comment]
I think this is probably obsoleted by more recent changes in the header, and an idea to allow choosing favorite tabs to show.  The issue this was responding to has been closed, so closing this, too.

**Pull request closed without merging**